croud/clusters/commands.py

```diff
@@ -622,6 +663,8 @@ def _wait_for_completed_operation(
                 "Our operations team are investigating."
             )
             break
+        if status == "IN_PROGRESS" and feedback_func:
```

Comment:
Otherwise does not print the final status:
```suggestion
        if status == "IN_PROGRESS" or status == 'SUCCEEDED'  and feedback_func:
```